### PR TITLE
Remove DataPointType and refactor

### PR DESCRIPTION
**Pull request by vishweshbankwar**

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public void ValidateMetricTelemetryItem()
             Assert.Equal(2, metricsData.Version);
             Assert.Equal("TestDoubleCounter", metricsData.Metrics.First().Name);
             Assert.Equal(123.45, metricsData.Metrics.First().Value);
-            Assert.Equal(DataPointType.Aggregation, metricsData.Metrics.First().DataPointType);
+            Assert.Null(metricsData.Metrics.First().DataPointType);
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:46]: Why asset on Null?

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:46]: I was going to ask the same question. Did we lose something with this refactor?

Comment (vishweshbankwar) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:46]: Removing `DataPointType` is intentional. I think asserting on it does not hurt. @TimothyMothra - not sure what you mean?

Comment (cijothomas) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:46]: this PR is not a refactor! it is intentionally removing DataPointType. (and doing refactoring too!)

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:46]: Sorry, ignore my comment, I missed that DataPointType was intentionally removed.

**Review by TimothyMothra — APPROVED** [review]

**Review by rajkumar-rangaraj — APPROVED** [review]

**Pull request merged**